### ISSUE TITLE
[Unexpected Behavior] GAMESTATE:GetGameSeed() is not random

### ISSUE THREAD

**Issue by JustinLove** [TeamRizu/OutFox#415]
**What you're using**

GAMESTATE:GetGameSeed()

**Expected behavior**

Based on former behavior and my understanding of it's purpose, the function should return a distinct seed for every "new game". The seed should be the same during songs within the same game, and change after game over. It should be unique each time a game is started (within the limits of pseudo random generators)

** Observed behavior **

The returned seed is not random after restarting the program, e.g. first game's seed is always 3922919429

**Comment by Scraticus** [comment]
Thanks for the report!

This was as I thought when we discussed in the discord server, the generator was being initialised always in the same way as it wasn't calling a random return, I've fixed this now so the game seed returns a random integer number as expected. The old behaviour for this is quite sketchy; it bases itself off of a really bad way to initialise a seed so I've moved away from that call completely.
